go.mod

```diff
@@ -26,3 +26,83 @@ require (
 	sigs.k8s.io/controller-runtime v0.9.6
 	sigs.k8s.io/yaml v1.2.0
 )
+
+require (
```

Comment:
As a reference why we have two require blocks: https://go.dev/doc/go1.17#go-command (that's new with 1.17)